config/init.go

```diff
@@ -138,6 +138,28 @@ func DefaultDatastoreConfig() Datastore {
 	}
 }
 
+func pebbleSpec() map[string]interface{} {
```

Comment:
@gammazero I wonder if we would have less maintenance in the future if we simplify things here a bit, and improve  UX:

-  not include config options with hardcoded zero values here, and only have them documented at https://github.com/ipfs/kubo/blob/master/docs/datastores.md
    - this way `ipfs init` does not include them in `config`, we can add new ones in the future, and we are ensuring the user who intents to change default at least had to open the datastores.md docs 
- move pebble configuration options (`pebble.Options`) to `map[string]interface{}` named `pebbleOptions`, and have it empty by default (`{}`) so users have place for adding options, but we don't prefill it with zero vals

Reply:
I completely agree with not including zero values. I removed them from the config profile.